web/src/components/SprungbrettListItem.tsx

```diff
@@ -1,23 +1,23 @@
 import * as React from 'react'
-import { ReactNode } from 'react'
+import { ReactElement } from 'react'
+import styled from 'styled-components'
 
 import { SprungbrettJobModel } from 'api-client'
 
 import ListItem from './ListItem'
 
+const Content = styled.div`
+  overflow-wrap: anywhere;
+`
+
 type PropsType = {
   job: SprungbrettJobModel
 }
 
-class SprungbrettListItem extends React.PureComponent<PropsType> {
-  render(): ReactNode {
-    const { job } = this.props
-    return (
-      <ListItem title={job.title} path={job.url}>
-        <div dir='auto'>{job.location}</div>
-      </ListItem>
-    )
-  }
-}
+const SprungbrettListItem = ({ job }: PropsType): ReactElement => (
+  <ListItem title={job.title} path={job.url}>
+    <Content dir='auto'>{job.location}</Content>
+  </ListItem>
+)
```

Comment:
Any reason to not keep `PureComponent` f.e. with
`export default memo(SprungbrettListItem)`
hard to measure performance but memo may increase the performance slightly

Reply:
I think on web we don't really have performance problems, especially for Sprungbrett since the list items are very simple. So I think we should be good without.